### ISSUE TITLE
Error: remove_overlap: Graphviz not built with triangulation library   in sfdp

### ISSUE THREAD

**Issue by trycmail** [ellson/graphviz#1269]
Hi,
I got a error message in  using sfdp command  of graphviz, "Error: remove_overlap: Graphviz not built with triangulation library", but the dot,fdp ,etc are perfect.
version: 2.41.338
platform: win7
download from: Appveyor( binary files.)
would you please help me.
Thanks a lot!

**Comment by emden** [comment]
It appears the Appveyor build for Windows does not check for either the GTS library or the Triangulation library. If one of these is available, the configuration process will set HAVE_GTS or HAVE_TRIANGLE and the appropriate code will be used.

One of these libraries is necessary to handle node overlap removal in the best way, using the Prism algorithm, so it is usually good to have one of them. As a workaround, you can run sfdp -Goverlap=true to avoid the call to  Prism. Also, fdp does use Prism, but only if there were node overlaps. its code does some overlap removal during layout, so for your graphs, the call to Prism may not be necessary.

**Comment by trycmail** [comment]
@emden Thanks for your reply,but the same error message is  even using then command: 
 sfdp -Goverlap=true
If there is another way?
Or fixing this bug in building Graphviz for windows platform?
thanks very much!

**Comment by magneticnorth** [comment]
> Or fixing this bug in building Graphviz for windows platform?
> 

Yes, it would be much better to build graphviz with libgts (Gnu Triangulated Surface Library)
as seen at http://gts.sourceforge.net <http://gts.sourceforge.net/>

**Comment by ErwinJanssen** [comment]
The reason GTS isn't added on Appveyor, is that I wasn't able to find a Windows version of the library. I could try manually compiling it and adding it to [the Windows dependency repository](https://github.com/ErwinJanssen/graphviz-windows-dependencies).

**Comment by twogee** [comment]
There seems to be a [Github project](https://github.com/finetjul/gts) that added CMake to GTS.

**Comment by emden** [comment]
I'm surprised that overlap=true didn't work. If not, that's a bug. I'll look into it.

In the old Windows package, I believe we built GTS, ANN and a couple of other libraries by hand, and made them part of the distribution.

**Comment by trycmail** [comment]
Thanks for all of your efforts.
Looking for good news about it.

**Comment by djw1809** [comment]
Hi -  I am having the same problem using graphviz on ubuntu 16.04 however i am calling sfdp from networkx via: 
`nx.drawing.nx_agraph.graphviz_layout(G,prog='sfdp')`

**Comment by peterjc** [comment]
Issue migrated to https://gitlab.com/graphviz/graphviz/issues/1269
